src/main/resources/assets/openblocks/lang/de_DE.lang

```diff
@@ -70,13 +86,24 @@ openblocks.misc.selected_cannon=Kanone zum Zielen ausgewählt
 openblocks.misc.pointed_cannon=Kanone zielt nun auf %s, %s, %s
 openblocks.misc.change_mode=Wechsle zu %s-Modus
 openblocks.misc.change_size=Ändere Größe zu %sx%sx%s
+openblocks.misc.change_box_size=Ändere Größe zu (%d,%d,%d):(%d,%d,%d)
 openblocks.misc.total_blocks=Insgesamt gezählte Blöcke: %d
 openblocks.misc.get_witched=Werde verhext!
-
-openblocks.misc.cant_restore=Das Inventar von Spieler %s kann nicht wiederhergestellt werden
-openblocks.misc.cant_store=Das Inventar von Spieler %s kann nicht gespeichert werden
-openblocks.misc.stored_inventory=Inventar des Spielers gespeichert in %s
+openblocks.misc.page=Seite %d von %d
+openblocks.misc.oh_no_ceiling=Du kannst hier nicht schlafen. Die Decke beunruhigt dich zu sehr...
+openblocks.misc.oh_no_ground=Auf dem DING willst du schlafen?!
```

Comment:
How about having it "Auf DEM Ding willst du schlafen ?!" Sounds a bit better, in my opinion.